rust/actyx/ax/tests/version_compat.rs

```diff
@@ -29,6 +27,27 @@ use std::{
 use tar::Archive;
 use tempfile::tempdir;
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(untagged)]
+#[allow(non_camel_case_types)]
+#[allow(clippy::upper_case_acronyms)]
+pub enum ActyxCliResult<T> {
```

Comment:
From what I tried and found, it isn't. To test internal components through code we need to expose `ax` as a library. Once published, it will be available for anyone to use but we can just "not document it" or tell people "you shouldn't be using me".